eZ/Publish/Core/settings/storage_engines/cache.yml

```
@@ -146,6 +146,12 @@ services:
         class: eZ\Publish\Core\Persistence\Cache\UrlWildcardHandler
         parent: ezpublish.spi.persistence.cache.abstractHandler
 
+    ezpublish.spi.persistence.cache.settingHandler:
```

at least
```suggestion
    ezpublish.spi.persistence.cache.setting_handler:
```
per [Symfony conventions](https://symfony.com/doc/current/contributing/code/standards.html#service-naming-conventions)
but keep in mind previous comment as well please.